packages/plugin-conversation/src/conversation.js

```diff
@@ -453,6 +453,24 @@ const Conversation = SparkPlugin.extend({
     return ShareActivity.create(conversation, null, this.spark);
   },
 
+  assign(conversation, avatar) {
+    return this._inferConversationUrl(conversation)
+      .then(() => {
+        const activity = ShareActivity.create(conversation, null, this.spark);
+        activity.add(avatar);
```

Comment:
this was the spec (similar to share activity but some props taken out) https://sqbu-github.cisco.com/WebExSquared/cloud-apps/wiki/Custom-Room-Avatar---Design-Spec